sdk/communication/Azure.Communication.Calling.Server/src/Models/Events/InviteParticipantsResultEvent.Serialization.cs

```diff
@@ -0,0 +1,61 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Text.Json;
+using Azure.Core;
+
+namespace Azure.Communication.Calling.Server
+{
+    public partial class InviteParticipantsResultEvent
+    {
+        /// <summary>
+        /// Deserialize <see cref="InviteParticipantsResultEvent"/> event.
+        /// </summary>
+        /// <param name="content">The json content.</param>
+        /// <returns></returns>
+        public static InviteParticipantsResultEvent Deserialize(string content)
```

Comment:
more manual code here

Reply:
These are added for handling events which users received on their callback which are outside SDK scope. We have to add manually because these models are not part of any request or response.